configs/dh-defaults.prop

```diff
@@ -133,7 +133,7 @@ Webapi.persistentquery.heap.default=4
 OneClick.allowedPattern.USym=^[A-Za-z0-9/.]+$
 OneClick.enforceUpperCaseColumns=USym
 
-LiveTableMonitor.checkTableOperations=true
+LiveTableMonitor.checkTableOperations=false
```

Comment:
Why is this necessary?

Reply:
This is to avoid a LTM exclusive lock check error when running the tests involving updating a time table in the test mode.

Reply:
This is to avoid a LTM exclusive lock check error when running the Python tests involving updating a time table in the test mode.

Reply:
This flag causes Python test cases on time table update etc to error out due to the exclusive lock check failure. This file is only used to set up the test environment for running standalone Python tests in Integrations and for py code generations in the Generators project. I have consulted @niloc132 and we both believe that it was a mistake to set it to true.

Reply:
@rcaudy do you think the change is warranted?

Reply:
test

Reply:
@rcaudy do you think the change is warranted?